changelog.md

```diff
@@ -1,3 +1,534 @@
+
+## Breaking changes
+
+### For programmatic users
+
+Under the hood, Lighthouse now uses the new user-flow supporting infrastructure by default, even for traditional navigation runs. You can opt out of this by: in the CLI, use `--legacy-navigation`; in DevTools: check “Legacy Navigation” in the settings menu. If you have a use case that necessitates this escape hatch, please file an issue. We plan to remove the legacy runner for 11.0.
+
+### For Lighthouse result JSON (LHR) users
+
+#### Page URLs on the Lighthouse Result
+
+Until now, there were two URL fields to describe a Lighthouse run:
+
+- `requestedUrl`: the url given by the users, which Lighthouse instructs Chrome to navigate to
+- `finalUrl`: the url after any server-initiated HTTP and JS-initiated redirects
+
+This taxonomy cannot account for more complex scenarios, such as JS-initiated redirects, usage of the History API or soft-navigations. They were also ill-defined for timespan and snapshot modes. To account for that, Lighthouse 10.0 now has these URL fields:
+
+- (changed) `requestedUrl`: The URL that Lighthouse initially navigated to before redirects. This is the same as it was before for navigation mode, but now it will be `undefined` in timespan/snapshot.
+- (new) `mainDocumentUrl`: The URL of the last document requested during a navigation. It does not account for soft navigations or history API events made after the page loads. It is only available in navigation mode, and will be undefined in timespan and snapshot modes.
+- (new) `finalDisplayedUrl`: The URL displayed in the browser combobox at the end of a Lighthouse run. It accounts for soft navigations and history API events. Available in navigation, timespan, and snapshot modes.
+- (deprecated) `finalUrl`: Same value as `mainDocumentUrl`.
+
+
+#### Audit changes
+
+- `password-inputs-can-be-pasted-into` -> `paste-preventing-inputs`
+- `preload-lcp-image` -> `prioritize-lcp-image`
+- `third-party-summary` no longer uses a `link` value for `item.entity`, instead uses a raw `text` value
+
+### For Node users
+
+- Node 14 is no longer supported, the minimum is now Node 16
+- In case you import paths within the lighthouse node package: `lighthouse-core/` and `lighthouse-cli/` folders are now simply `core/` and `cli/`
+- Converted from CommonJS to ES modules. For access to just the `lighthouse` function in CommonJS, you can use `require('lighthouse/core/index.cjs')`
+- The CSV output for Lighthouse is much more useful now. Consult the PR for [an example of the new format](https://github.com/GoogleChrome/lighthouse/pull/13558)
+- `LHError` is now `LighthouseError`. If you are attempting to catch an error thrown by Lighthouse, be sure to account for this!
+
+#### Node API changes
+
+The `lighthouse` function now has [better integration with Puppeteer](https://github.com/GoogleChrome/lighthouse/blob/main/docs/puppeteer.md). Use `lighthouse(url, flags, config, page)` to run Lighthouse in an existing Puppeteer environment.
```

Comment:
any way to indicate here that `page` is a puppeteer `Page`? it's not completely clear just from reading it as is what the better integration is